packages/@ourworldindata/core-table/src/OwidTable.ts

```diff
@@ -319,6 +319,64 @@ export class OwidTable extends CoreTable<OwidRow, OwidColumnDef> {
         )
     }
 
+    /**
+     * Drop _all rows_ for an entity if the entity has no data in ALL of the specified columns.
+     * An entity is kept if it has at least one valid value in ANY of the columns.
+     */
+    dropEntitiesThatHaveNoDataInAllColumns(columnSlugs: ColumnSlug[]): this {
```

Comment:
How is this different from `dropRowsWithErrorValuesForAllColumns`? Is it at all different?

I tried changing the test to use `dropRowsWithErrorValuesForAllColumns` instead, and it still passed.